### DISCUSSION TITLE
Zigbee gear does not go into pairing mode

### DISCUSSION THREAD

**Discussion by phbender** [Koenkk/zigbee2mqtt#14408] [Devices]
Hi,

I have three brand new [Sonoff wireless switches](https://www.zigbee2mqtt.io/devices/SNZB-01.html) and can't get them to pair.

zigbee2mqtt: 1.28.0
Coordinator: zStack12
Coordinator Revision: 20190608

Here is what I did:

- make sure zigbee2mqtt allows joining (tried setting it in the YAML as well as in the front end)
- Reset the switch like described in the [docs](https://www.zigbee2mqtt.io/devices/SNZB-01.html)
- Place the switch near the coordinator

I expected the switch to show up either in the frond end, in the logs or in `configuration.yaml`.

I tried with all three switches that I got just to make sure it's not a single bad battery or some such.

What can I do to debug? The switch is reported to work, and I found posts that had issues with it, but not while pairing.

Thanks,
Philipp

**Comment by phbender** [comment]
I have stopped my container and started from clean slate (generate a new configuration etc.). This time, the interview started and the device was correctly paired.
I will now try to re-set the other devices and find out what in the old configuration the problem could have been.

**Comment by phbender** [comment]
After purchasing an additional set of Sonoffs, I ran into the same problem agian: They did not pair.
This time, I was successful with completely re-starting the whole thing:

- `docker-compose down`
- unplug the zigbee stick, wait 10 seconds
- plug the stick in again
- `docker-compose up`

Then, after just a few seconds, the devices started to pair. Solely restarting the containers did not, so I assume it's a hardware issues as well.

**Comment by desertblade** [comment]
I would recommend disabling permit_join: true from the configuration and the UI.  I don't use that setting and if Z2M allowed random devices to join all the time it would be a security risk. I am guessing on restart it goes into "Allow Join" mode for a few minutes then stops.

In my environment I click on "Permit Join (All)" in the top menu, then place the device in pairing in pairing mode.

**Comment by phbender** [comment]
In the meantime, my setup is working reliable again. I had a bunch of issues which were solved by switching by dongle, I bought a Sonoff one and replaced the one I had which I saw is also listed in the documentation of being not reliable (don't have the name at the moment).
